src/shared/components/table/Table.scss

```diff
@@ -0,0 +1,17 @@
+@import 'styles/mixins';
+
+.table {
+  display: grid;
+  font-size: 14px;
+  width: 1345px;
+
+  .header {
```

Comment:
Where is .header located. I checked several files and no item with this class I think this can be removed